<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportAnimationTool_Window</class>
 <widget class="QMainWindow" name="ExportAnimationTool_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Anim Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>File Path</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="ExportAnimTool_filePath"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="ExportAnimTool_filePathButton">
        <property name="text">
         <string>...output</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>NameSpace</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="ExportAnimTool_NameSpaceString">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Export Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ExportAnimTool_ExportTypeAll">
           <property name="text">
            <string>All</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ExportAnimTool_ExportTypeSelect">
           <property name="text">
            <string>Select</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ExportAnimTool_ExportTypeHierarchy">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Hierarchy</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Create</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>catch Cmmand</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="ExportAnimTool_CatchToggelTrue">
           <property name="text">
            <string>True</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ExportAnimTool_CatchToggelFalse">
           <property name="text">
            <string>False</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Notice(通知)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="ExportAnimTool_notice">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="ExportAnimTool_Run">
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ExportAnimTool_ExportTypeAll</sender>
   <signal>toggled(bool)</signal>
   <receiver>ExportAnimTool_NameSpaceString</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
